opencti-platform/opencti-front/src/private/components/analyses/notes/NoteCreation.tsx

```diff
@@ -126,19 +126,21 @@ export const NoteCreationForm: FunctionComponent<NoteFormProps> = ({
   const classes = useStyles();
   const { t_i18n } = useFormatter();
   const userIsKnowledgeEditor = useGranted([KNOWLEDGE_KNUPDATE]);
-  const basicShape = {
+  const { mandatoryAttributes: mandatoryAttributesBase } = useIsMandatoryAttribute(NOTE_TYPE);
+  // created is not managed in the DB Schema, but is required.
+  const mandatoryAttributes = mandatoryAttributesBase.concat(['created']);
```

Comment:
you can remove this concat because `created` is now managed, so in this case we create a duplicate